### ISSUE TITLE
Change field type to `:text` for the item text

### ISSUE THREAD

**Issue by SimonLab** [dwyl/mvp#204]
The application can fail to create a new item when the text content is too long:

```
(Postgrex.Error) ERROR 22001 (string_data_right_truncation) value too long for type character varying(255)
```

This is due to the migration defining the text value as `:string`:
https://github.com/dwyl/mvp/blob/f5bd6f1880378260091de62fa7528f8753649cb9/priv/repo/migrations/20220627162154_create_items.exs#L6

- [ ] Create a new migration to alter the `items` table to define the field with the `:text` type instead of `:string`

```elixir
alter table(:items) do
  modify :text, :text
end
```

**Comment by nelsonic** [comment]
Also: 
https://github.com/dwyl/mvp/blob/f5bd6f1880378260091de62fa7528f8753649cb9/lib/app/item.ex#L10

**Comment by SimonLab** [comment]
Ecto is using the type `:sring` in the schema, see:
https://github.com/dwyl/mvp/pull/205#issuecomment-1314039013

**Comment by SimonLab** [comment]
Fixed with #205

**Comment by nelsonic** [comment]
Thanks @SimonLab 👌
